xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/XWiki/DeletedAttachments.xml

```diff
@@ -153,6 +154,9 @@ $response.setContentType('application/json')
 #if("$!{docNameFilter}" != '')
   #set($sql = "and lower(datt.docName) like '%${docNameFilter.trim().toLowerCase()}%' ")
 #end
+#if("$!{dateFilter}" != '')
+  #set($sql = $sql + "and ddoc.date between '%${dateFilter.split("-")[0]}%' and '%${dateFilter.split("-")[1]}%' ")
```

Comment:
This `ddoc` does not seems to be defined anywhere.

Reply:
Look for the ``$selectSql `` variable below. There's no ``ddoc`` alias defined. I see only the ``datt`` alias (because the query selects deleted attachments, not deleted documents). It looks like you copied the changes from the ``XWiki.DeletedDocumentsJSON`` page without adapting them and without testing your changes...

Reply:
Also, it would be nice to use query parameters instead of appending the filter value directly to the query, in order to prevent SQL injection. You can find examples on the [Query Module documentation](http://extensions.xwiki.org/xwiki/bin/view/Extension/Query+Module).